cdc/sink/common/common_test.go

```diff
@@ -29,70 +31,131 @@ var _ = check.Suite(&SinkCommonSuite{})
 
 func (s SinkCommonSuite) TestSplitResolvedTxn(c *check.C) {
 	defer testleak.AfterTest(c)()
-	testCases := []struct {
-		unresolvedTxns         map[model.TableID][]*model.SingleTableTxn
-		resolvedTs             uint64
-		expectedResolvedTxns   map[model.TableID][]*model.SingleTableTxn
-		expectedUnresolvedTxns map[model.TableID][]*model.SingleTableTxn
-		expectedMinTs          uint64
-	}{{
-		unresolvedTxns: map[model.TableID][]*model.SingleTableTxn{
-			1: {{CommitTs: 11}, {CommitTs: 21}, {CommitTs: 21}, {CommitTs: 23}, {CommitTs: 33}, {CommitTs: 34}},
-			2: {{CommitTs: 23}, {CommitTs: 24}, {CommitTs: 26}, {CommitTs: 26}, {CommitTs: 26}, {CommitTs: 29}},
+	testCases := [][]struct {
```

Comment:
Please add comment to the test goal of each case

Reply:
PTAL again